src/main/resources/assets/ironbackpacks/lang/zh_CN.lang

```diff
@@ -1,5 +1,43 @@
 #PARSE_ESCAPES
 
+# Creative Tab
+itemGroup.ironbackpacks=特殊背包
+
+# Items
+item.ironbackpacks.backpack.name=破损背包
+item.ironbackpacks.backpack.ironbackpacks.basic.name=基础背包
+item.ironbackpacks.backpack.ironbackpacks.iron.name=铁背包
+item.ironbackpacks.backpack.ironbackpacks.gold.name=金背包
+item.ironbackpacks.backpack.ironbackpacks.diamond.name=钻石背包
+item.ironbackpacks.upgrade.name=空白升级
+
+# Tooltips
+tooltip.ironbackpacks.upgrade.cost=需要消耗%d升级点数（s）
```

Comment:
Chinese does not have singular / plural. Remove `（s）`. Also add space between Arabic numerals and Chinese characters.